### ISSUE TITLE
Saving in Camarina Database not possible again

### ISSUE THREAD

**Issue by ReKlug** [DH-Center-Tuebingen/Spacialist#417]
It is the same problem like last week, but even worth. Now spacialist is saying that the changes are saved, but they are not. Even more Entities seem to be corrupted.
The problem started, after creating new attributes in the thesaurus tool (last week) or creating new connections between the attributes (today).

**Comment by v1r0x** [comment]
I'm sorry, but I can't confirm this. I tried to edit several entites (and several entity types). Everything worked fine.
If there is still something wrong on your side, please give a step by step guide to reproduce this error.

**Comment by ReKlug** [comment]
I am really sorry, but the problem is still the same. 
I checked it right now: I tried to edit the subentity SF 11 – Fiume Grande SP 3 (Tree: Vizzini – Fiume Grande – Fiume Grande – SF 11). I added “Afrikanische Amphore” to the table “Prospektionsfunde” and saved it. The database said it was saved. Then I opened SF 11 – Fiume Grande SP 2. I did not change anything there. After this I went back to SP 3, and there was no “Afrikanische Amphore”.

I recorded all my steps.

Greetings Rebecca
[Spacialist.zip](https://github.com/DH-Center-Tuebingen/Spacialist/files/10765874/Spacialist.zip)
